css-transforms/css-transforms-quiz-notes.md

```diff
@@ -0,0 +1,39 @@
+# css-transforms-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What does the transform property do?
+
+- Give four examples of CSS transform functions.
```

Comment:
Please make sure to answer the quiz questions!

Reply:
Okay, I added the answers to the quiz questions. Thanks!